src/ok.rs

```diff
@@ -58,6 +62,40 @@ impl fmt::Display for TestState {
     }
 }
 
+enum SnpIoctl {
+    Tcb,
+    Rmp,
+    Snp,
+}
+
+enum SevIoctl {
+    Sev,
+    Firmware,
+    Seves,
+}
+
+impl fmt::Display for SnpIoctl {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let s = match self {
+            SnpIoctl::Tcb => format!("tcb"),
+            SnpIoctl::Rmp => format!("rmp"),
+            SnpIoctl::Snp => format!("snp"),
+        };
+        write!(f, "{}", s)
+    }
+}
+
+impl fmt::Display for SevIoctl {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let s = match self {
+            SevIoctl::Sev => format!("sev"),
+            SevIoctl::Firmware => format!("firmware"),
+            SevIoctl::Seves => format!("seves"),
+        };
+        write!(f, "{}", s)
+    }
+}
+
```

Comment:
Here in the display you can use it to display the actual message/name of the test!

So later on in the code you can do:

`TestResult {
                    name: format!("{}", SnpIoctl::Snp),
                    stat: TestState::Pass,
                    mesg: format!("SNP is INIT").into(),
                }`

Instead of:

`TestResult {
                    name: "SNP INIT".to_string(),
                    stat: TestState::Pass,
                    mesg: format!("SNP is INIT").into(),
                }`